sjsonnet/src/sjsonnet/Val.scala

```diff
@@ -297,15 +298,45 @@ object Val{
     }
   }
 
-  def staticObject(pos: Position, fields: Array[Expr.Member.Field]): Obj = {
+  final case class StaticObjectFieldSet(keys: Array[String]) {
```

Comment:
This shouldn't be a case class. `Array` has no useful equality or `toString` and you're overriding `equals` and `hashCode`.

Reply:
Fixed